br/pkg/restore/client.go

```diff
@@ -1206,6 +1215,12 @@ func (rc *Client) RestoreSSTFiles(
 		)
 		return errors.Trace(err)
 	}
+	// Once the parent context canceled and there is no task running in the errgroup,
+	// we may break the for loop without error in the errgroup. (Will this happen?)
+	// At that time, return the error in the context here.
+	if err := ectx.Err(); err != nil {
```

Comment:
I'm not understand when will this error happen. what if we just return nil?

Reply:
> what if we just return nil?
The user may get some confusing error message.